AGENT_CHANGELOG.md

```diff
@@ -1,8 +1,46 @@
-## Datadog Agent version [7.49.1](https://github.com/DataDog/datadog-agent/blob/master/CHANGELOG.rst#7491)
-
-* Datadog Checks Base [34.0.3](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/CHANGELOG.md)
+## Datadog Agent version [7.50.0](https://github.com/DataDog/datadog-agent/blob/master/CHANGELOG.rst#7500)
+
+* Amazon Kafka [4.3.0](https://github.com/DataDog/integrations-core/blob/master/amazon_msk/CHANGELOG.md)
+* Ambari [4.0.0](https://github.com/DataDog/integrations-core/blob/master/ambari/CHANGELOG.md) **BREAKING CHANGE**
+* Cacti [2.0.1](https://github.com/DataDog/integrations-core/blob/master/cacti/CHANGELOG.md)
+* Datadog Checks Base [34.3.1](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/CHANGELOG.md)
```

Comment:
Hmm, this should be 34.3.0

Reply:
Double-checked, 34.3.1 is correct, we [bumped cryptography in this agent release](https://github.com/DataDog/integrations-core/pull/16322)